### DISCUSSION TITLE
FLIF support, either by GDNative or by other means..

### DISCUSSION THREAD

**Discussion by anderlli0053** [godotengine/godot-proposals#4621] [General Discussions]
Maybe someone will add support for FLIF n the future?? be it as a module or whatever...

Just opening this to see who else is even interested in FLIF file format... beside me..

_and please don't start with "maybe if you pay someone to make the module, ..." yeah, that's s just not going to happen from my side... i'm just curious, if anyone else is interested in the FLIF and its integration in Godot._

http://flif.info/

**Comment by JoNax97** [comment]
From the first paragraph of the page you linked:

_FLIF development has stopped since FLIF is superseded by FUIF and then again by [JPEG XL](https://jpeg.org/jpegxl),_

Not sure about this one.

**Reply by anderlli0053** [reply]
I know, i'm just curious if anyone is using that format, that is outside Godot and all

**Comment by Calinou** [comment]
See also https://github.com/godotengine/godot-proposals/issues/2461.

That said, Godot already has WebP support (lossless and lossy). WebP is good enough for most use cases already :slightly_smiling_face:
